src/collective/behavior/localdiazo/behavior.py

```diff
@@ -34,4 +33,4 @@ class ILocalDiazo(ILocalRegistry):
         required=True
     )
 
-alsoProvides(ILocalDiazo, form.IFormFieldProvider)
+alsoProvides(ILocalDiazo, IFormFieldProvider)
```

Comment:
check how this is implemented: https://github.com/collective/sc.social.like/blob/2.13b1/sc/social/like/behaviors.py